security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -116,6 +117,20 @@ type sdsservice struct {
 	closing chan bool
 }
 
+type sdsclientdebug struct {
+	ConnectionID string `json:"connection_id"`
```

Comment:
Are these fields needed to be public?

Reply:
To use `json.Marshal`, the fields have to be exported (b/c reflection). I could do something like what's described [here](https://stackoverflow.com/questions/11126793/json-and-dealing-with-unexported-fields) as well